### DISCUSSION TITLE
Tailwindcss and WordPress

### DISCUSSION THREAD

**Discussion by iamdarzee** [tailwindlabs/tailwindcss#9506] [Help]
Hi, so I created some custom width and height in the tailwind.config.js file, as follows:
/** @type {import('tailwindcss').Config} */
module.exports = {
content: ['./*.html'],
  theme: {
    extend: {
      width: {
        '600': '600px',
        '900': '900px',
      height: {
        '800': '800px',
        '400': '400px',
    },
  },
  plugins: [],
}

When I use h-400 when making the site normally, vs code + chrome, everything works fine But now when making a template inside WordPress using the same HTML and adding PHP the custom tailwindcss like h-800 or w-900 don't work even some colors sometimes, so this means what I build is different from what is Live.

TIA.

**Comment by wongjn** [comment]
You may need to look at updating your `content` configuration to sufficiently cover your source files that contain your Tailwind classes. So if you have PHP files in a `templates` directory for example, you could add `templates/**/*.php`. At the moment, the `content` glob only covers a single file named `.html` in the project root.
